phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/ViewsConfigurationLibrary.java

```diff
@@ -39,7 +39,11 @@
 
     @Bean
     public ViewLibrary viewLibrary() {
-        return new ViewLibrary(applicationName, themeConfiguration.themeManager(), phoenicisGlobalConfiguration.objectMapper());
+        return new ViewLibrary(
```

Comment:
Did the plugin work for you with `mvn test` or did it at least give you a different formatting?